tests/account_change_password_test.go

```diff
@@ -0,0 +1,48 @@
+package tests
+
+import (
+	"testing"
+
+	"encoding/json"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+func TestChangePassword(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintAlice)
+	defer deployment.Destroy(t)
+	unauthedClient := deployment.Client(t, "hs1", "")
+	newPassword := "new_superuser"
+	// sytest: After changing password, can't log in with old password
+	t.Run("After changing password, can't log in with old password", func(t *testing.T) {
+		passwordClient := deployment.RegisterUser(t, "hs1", "cant_login_with_old_password", "superuser")
+		reqBody := client.WithJSONBody(t, map[string]interface{}{
+			"auth": map[string]interface{}{
+				"type":         "m.login.password",
+				"user":         "@cant_login_with_old_password:hs1",
```

Comment:
Prefer using `passwordClient.UserID` here so we don't have to keep updating these strings if the user ID changes.